.madrun.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+function dockerPush(type, version = 'latest') {
+    return `docker push coderaiser/cloudcmd:${version}-${type}`;
+}
+
```

Comment:
Could you please move this function at the bottom of a file :)

Reply:
done ✅